src/Playwright/Transport/Connection.cs

```diff
@@ -284,7 +279,7 @@ private void Transport_MessageReceived(object sender, MessageReceivedEventArgs e
                 return;
             }
 
-            _logger?.LogInformation($"pw:channel:event {e.Message}");
+            _logger?.LogTrace($"pw:channel:event {e.Message}");
```

Comment:
in Python we have something like that:

```
if process.env.DEBUGP === "1":
  console.log(message)
```
for messages which we receive and send. I would recommend doing it too in .NET since this should never be interesting for the user.